twoliter/src/cmd/exec.rs

```diff
@@ -0,0 +1,107 @@
+use crate::common::exec;
+use crate::project;
+use crate::tools::install_tools;
+use anyhow::Result;
+use clap::Parser;
+use log::trace;
+use std::path::PathBuf;
+use tokio::process::Command;
+
+/// Run a cargo make command in Twoliter's build environment. Known Makefile.toml environment
+/// variables will be passed-through to the cargo make invocation.
```

Comment:
Should this command be called `twoliter make`? Since it ends up being pretty specific to `cargo-make` and `Makefile.toml`.